backend/app.js

```diff
@@ -43,16 +69,28 @@ app.get('/users/:id', async (req, res) => {
 })
 
 // Demo for registering new user
-app.post('/register', async (req, res) => {
-  try {
-    console.log(req.body)
-    const user = await User.create(req.body)
-    res.status(200).json(user)
-  } catch (error) {
-    console.log(error.message)
-    res.status(500).json({ message: error.message })
-  }
-})
+// app.post('/register', async (req, res) => {
```

Comment:
Make sure to clean up all commented out code unless it is super important to be kept for the functionality of future sprints :) (helps prevent clutter within the files as they get bigger)